src/cljs/rems/application_list.cljs

```diff
@@ -58,6 +58,12 @@
                           {:class (when (application-util/form-fields-editable? app)
                                     "text-highlight")}
                           value]})
+           :todo (let [value (localize-todo (:application/todo app))]
+                   {:value value
+                    :td [:td.state
+                         {:class (when (str/includes? (str (:application/todo app)) "-your-")
+                                   "text-highlight")}
```

Comment:
It might not be a problem of this patch (as a similar means for highlighting is used elsewhere), but I think the grey color alone doesn't work very well for highlighting the text. How about using bold in addition? See screenshots below.

![image](https://user-images.githubusercontent.com/49641103/64536758-56325a00-d322-11e9-96ee-5387114afac3.png)

and in a context:

![image](https://user-images.githubusercontent.com/49641103/64536676-2be09c80-d322-11e9-8f14-492449a90a11.png)